config.TEMPLATE.inc.php

```diff
@@ -577,6 +572,38 @@
 ; Remove this setting to leave failed jobs in the database.
 delete_failed_jobs_after = 180
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Schedule Task Settings ;
+;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+[schedule]
+
+; Whether or not to turn on the built-in schedule task runner
+;
+; When enabled, schedule tasks will be processed at the end of each web
+; request to the application.
+;
+; Use of the built-in schedule task runner is highly discouraged for high-volume 
+; sites. Instead, system's crontab should be configured to run every minutes as 
+; * * * * * cd /path-to-your-project && php lib/pkp/tools/scheduler.php run >> /dev/null 2>&1
```

Comment:
I removed the `cd`, because the commands are expected to know where they are running.

```suggestion
; Sample for the *nix crontab
; * * * * * php lib/pkp/tools/scheduler.php run >> /dev/null 2>&1
```